src/Symfony/Bundle/FrameworkBundle/EventListener/TemplateListener.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\EventListener;
+
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Bundle\FrameworkBundle\Templating\TemplatedResponseInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+class TemplateListener implements EventSubscriberInterface
+{
+    private $templating;
+
+    public function __construct(EngineInterface $templating)
```

Comment:
having a hard dependency to the templating component is not fine IMO, as we worked very hard to make it possible to use Twig natively without wrapping it in the Templating component.